src/newrelic/harvest.go

```diff
@@ -48,6 +49,11 @@ func NewHarvest(now time.Time, hl collector.EventConfigs) *Harvest {
 		pidSet:            make(map[int]struct{}),
 		httpErrorSet:      make(map[int]float64),
 	}
+
+	// MSF - POSSIBLE REMOVE FROM RELEASE
+	log.Debugf("NewHavest() created with log limits = %d", hl.LogEventConfig.Limit)
+
```

Comment:
Should probably remove this

```suggestion
```

Reply:
This change is in commit [a198099](https://github.com/newrelic/newrelic-php-agent/pull/497/commits/a198099168e2b57511fb842827feba16138366d4).